addons/account/models/chart_template.py

```diff
@@ -42,12 +42,19 @@ def _remove_xml_id(xml_id):
             module, name = xml_id.split(".", 1)
             env['ir.model.data'].search([('module', '=', module), ('name', '=', name)]).unlink()
 
+        def _avoid_name_conflict():
+            conflict_tax = env['account.tax'].search([('name', '=', template.name), ('company_id', '=', company.id),
+                                                      ('type_tax_use', '=', template.type_tax_use), ('tax_scope', '=', template.tax_scope)])
+            if conflict_tax:
+                conflict_tax.name = "[old] " + conflict_tax.name
```

Comment:
We're going to break the translations for l10n_multilang modules with this, but I guess this should only happen very rarely and there's no other choice.